### PR TITLE
Build CKEditor only for PRs. Run dev tests for all branches.

### PR DESCRIPTION
**Pull request by f1ames**
The result of above `.travis.yml` script modifications:
- Dev tests are run for all branches according to Travis settings.
- Build tests are run only for PRs.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,33 +23,40 @@ addons:
 before_script:
   - 'npm install benderjs-cli -g'
 
+  # Build CKEditor only for master, major branches and PRs (since every change eventually goes through PR process).
+  - 'RUN_BUILD=0'
+  - 'if [ "$BUILD" = "1" ] && [ "$TRAVIS_PULL_REQUEST" != "false" ]; then RUN_BUILD=1; fi'
+  - 'if [ "$TRAVIS_BRANCH" = "master" ] || [ "$TRAVIS_BRANCH" = "major" ]; then RUN_BUILD=1; fi'
```
Comment (Comandeer) [inline .travis.yml]: If I think correctly, this `if` causes to run tests only on built version of CKE on `master` and `major` branches.

**Comment by mlewand** [comment]
Guys, just one question, could you take this chance to extract this script into a standalone script file? Having multiple bash lines it in a `yml` file is a bit problematic.

**Comment by f1ames** [comment]
Extracted build procedure to separate files, now `.travis.yml` looks much better.

**Comment by Comandeer** [comment]
@f1ames could you rebase this branch to master and merge?

**Comment by f1ames** [comment]
Merged with https://github.com/ckeditor/ckeditor-dev/commit/efcee835265e1455c6c8ba58d8215de3a553ff82. I did not want to create separate PR just to merge it so I did it manually as this PR targeted `major`.

**Pull request closed without merging**